capa/rules.py

```diff
@@ -568,6 +571,18 @@ def build_statements(d, scope: str):
         or (key == "arch" and d[key] not in capa.features.common.VALID_ARCH)
     ):
         raise InvalidRule("unexpected %s value %s" % (key, d[key]))
+
+    elif key.startswith("property/"):
+        access = key[len("property/") :]
```

Comment:
assert valid access here? or wait for exception in `Property` constructor.

Reply:
done